SYCL/USM/copy.cpp

```diff
@@ -71,6 +71,10 @@ template <typename T> void test(queue q, T val, T *src, T *dst, bool dev_dst) {
 }
 
 template <typename T> void runTests(queue q, T val, alloc kind1, alloc kind2) {
+  if ((std::is_same_v<T, double> || std::is_same_v<T, test_struct>)&&!q
```

Comment:
Maybe we can just remove `double` from the `test_struct` or have two structs - with and without double.

Reply:
I have applied this suggestion in #1242